src/components/ivq-bottom-bar/ivq-bottom-bar.tsx

```diff
@@ -1,33 +1,51 @@
 import {h, VNode} from 'preact';
-import {useMemo, useCallback, useEffect, useState} from 'preact/hooks';
+import {useMemo, useEffect, useState} from 'preact/hooks';
 import {icons} from '../icons';
+import {QuizTranslates} from '../../types';
 import * as styles from './ivq-bottom-bar.scss';
 
 const {SeekBarPlaybackContainer, Icon} = KalturaPlayer.ui.components;
+const {withText, Text} = KalturaPlayer.ui.preacti18n;
 
 interface IvqBottomBarProps {
   onPrev?: () => void;
   onNext?: () => void;
   questionCounter: string | VNode<{}>;
 }
 
-export const IvqBottomBar = ({onPrev, onNext, questionCounter}: IvqBottomBarProps) => {
+const translates: QuizTranslates = {
+  nextQuestionButtonAriaLabel: <Text id="ivq.next_question_button_area_label">Next Question button</Text>,
```

Comment:
No need to write button in the area label. The screen reader knows it is a button if the tag is button or it has a role of a button